stock_report_deliveryslip_adj/README.rst

```diff
@@ -14,12 +14,16 @@ Stock Report Delivery Slip Adjustments
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-qrtl%2Fhls--custom-lightgray.png?logo=github
-    :target: https://github.com/qrtl/hls-custom/tree/12.0/stock_report_deliveryslip_adj
+    :target: https://github.com/qrtl/hls-custom/tree/4419-fix-stock_report_deliveryslip_adj/stock_report_deliveryslip_adj
```

Comment:
You need to run `oca-gen-addon-readme` with `--branch=12.0`, not your feature branch.

```suggestion
    :target: https://github.com/qrtl/hls-custom/tree/12.0/stock_report_deliveryslip_adj
```